docs/source/ec2.rst

```diff
@@ -1 +1,4 @@
+Cloud Deployment
+=================
+
 See `Dask's cloud deployment documentation <https://dask.pydata.org/en/latest/setup/cloud.html>`_ for up-to-date documentation for deployment on Amazon's Cloud.
```

Comment:
We no longer publish this file.  It's only here in order to redirect users.  

Probably we should just use a proper redirect.  There is a dictionary of redirects in `source/conf.py` that might be good for this.